package.json

```diff
@@ -45,30 +45,31 @@
   ],
   "preferGlobal": true,
   "dependencies": {
-    "chalk": "^4.1.2",
-    "commander": "^7.2.0",
-    "fs-extra": "^10.1.0",
-    "hap-nodejs": "~0.11.1",
-    "qrcode-terminal": "^0.12.0",
-    "semver": "^7.5.4",
-    "source-map-support": "^0.5.21"
+    "chalk": "4.1.2",
```

Comment:
Why are we migrating to fixed versions for all dependencies? Shouldn’t we leave it as „up to next major“ or at least „up to next minor“?

Reply:
i've taken this approach in the UI repo just for stability, I personally prefer this approach, but since the homebridge repo is not my domain, happy to revert to ^ or ~.

@NorthernMan54 any preference?

Reply:
@Supereg  My preference is not to have support issues, so I lean towards fixed versions.  As @bwp91 mentions we have been hit a couple of times by dependancy updates breaking things.

And for example last week I broke Scrypted when updating node-pty-prebuilt-multiarch, with a change that their configuration couldn't handle - https://github.com/homebridge/node-pty-prebuilt-multiarch/pull/41 .

Reply:
Well fair, ^ or ~ built around the trust that dependencies adhere to semantic versioning.
I'm also fine with fixing versions. 👍